dmipy/core/modeling_framework.py

```diff
@@ -883,6 +883,8 @@ def _construct_convolution_kernel(self, **kwargs):
 
         Parameters
         ----------
+        acquisition_scheme: DmipyAcquisitionScheme instance,
+            An acquisition scheme that has been instantiated using Dmipy.
         x0_vector: array of size (N_parameters),
```

Comment:
please remove this deprecated `x0_vector`

Reply:
And please change the function signature to `def _construct_convolution_kernel(self, acquisition_scheme, **kwargs):`